binding/lua/docs/API.md

```diff
@@ -1,12 +1,15 @@
 # Multiverso Torch Binding API
 
-## init()
+## init(sync)
 
 Initialize mutliverso.
 
 This should be called only once before training at the beginning of the whole
 project.
 
+If sync is True, a sync server will be created. Otherwise an async server
```

Comment:
Hummm, as it's the document for `Lua` binding, I think we should use `true` instead of `True`. It will be better if we also have the code block there for highlight.